From 56072d7282e7daf65f58fa897f1f76268ae88121 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 6 Feb 2016 11:06:26 -0800
Subject: [PATCH 16/16] define PROJECT_APIVER

this helps in compiling components which use packageconfig to poke for
library versions and requiring minimum supported version

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 CMakeLists.txt | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b240ef6..aa5e14f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 2.8)
 
 project(vmcs_host_apps)
 
+SET(PROJECT_VER_MAJOR 1)
+SET(PROJECT_VER_MINOR 0)
+SET(PROJECT_VER_PATCH 0)
+SET(PROJECT_VER "${PROJECT_VER_MAJOR}.${PROJECT_VER_MINOR}.${PROJECT_VER_PATCH}")
+SET(PROJECT_APIVER "${PROJECT_VER}")
+
 set(BUILD_MMAL TRUE)
 set(BUILD_MMAL_APPS TRUE)
 set(vmcs_root ${PROJECT_SOURCE_DIR})
-- 
2.7.0

